NPE

**Describe the bug**
Getting an error:
```
[12:54:39] [Netty Worker IO Thread #13/ERROR]: [Protocolize] === EXCEPTION CAUGHT IN DECODER ===
[12:54:39] [Netty Worker IO Thread #13/ERROR]: [Protocolize] Stream: UPSTREAM
[12:54:39] [Netty Worker IO Thread #13/ERROR]: [Protocolize] Connection: [/player ip |player name] <-> InitialHandler
[12:54:39] [Netty Worker IO Thread #13/ERROR]: [Protocolize] Protocol version: 756
[12:54:39] [Netty Worker IO Thread #13/ERROR]: io.netty.handler.codec.EncoderException: java.lang.NullPointerException: Cannot invoke "de.exceptionflug.protocolize.items.ItemStack.write(io.netty.buffer.ByteBuf, int)" because "this.cursorItem" is null
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:125)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:709)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:792)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:702)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:110)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at java.base/java.lang.Thread.run(Thread.java:831)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: Caused by: java.lang.NullPointerException: Cannot invoke "de.exceptionflug.protocolize.items.ItemStack.write(io.netty.buffer.ByteBuf, int)" because "this.cursorItem" is null
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at de.exceptionflug.protocolize.items.packet.WindowItems.write(WindowItems.java:90)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at net.md_5.bungee.protocol.MinecraftEncoder.encode(MinecraftEncoder.java:24)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at net.md_5.bungee.protocol.MinecraftEncoder.encode(MinecraftEncoder.java:9)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107)
[12:54:39] [Netty Worker IO Thread #13/ERROR]: 	... 14 more

```

**To Reproduce**
N/A (idk)

**Expected behavior**
No errors

**Proxy environment**
paper 448
**Minecraft versions used**
paper 1.17.1 build 196

--- a/protocolize-items/src/main/java/de/exceptionflug/protocolize/items/packet/WindowItems.java
+++ b/protocolize-items/src/main/java/de/exceptionflug/protocolize/items/packet/WindowItems.java
@@ -87,7 +87,9 @@ public class WindowItems extends AbstractPacket {
       item.write(buf, protocolVersion);
     }
     if (protocolVersion >= MINECRAFT_1_17_1) {
-      cursorItem.write(buf, protocolVersion);
+      if (cursorItem != null) {
+        cursorItem.write(buf, protocolVersion);
+      }
     }
   }
 

